src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -127,10 +125,14 @@ private void validateFullCommonDetails(OverseasEntitySubmissionDto overseasEntit
     private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         if(isTrustWebEnabled &&
                 !CollectionUtils.isEmpty(overseasEntitySubmissionDto.getTrusts())) {
-            trustDetailsValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-            trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-            historicalBeneficialOwnerValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-            trustCorporateValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
+            if(!overseasEntitySubmissionDto.isForUpdate()) {
+                trustDetailsValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
```

Comment:
move line 129 outside of if/then/else. Then you only need it once! And no else.

Reply:
done